DatabaseManager/DatabaseManager.cs

```diff
@@ -1331,7 +1334,7 @@ public static SqlDataReader CreateDataReader( String strQuery, ConnectionParamet
                 throw new Exception( "Attempted to get MSSQL datareader for OleDB connection." );
```

Comment:
There is some funky spacing going on here.

Reply:
I didn't run code maid on this file because it will change spacing throughout the file.
Should I do it?